docs/framework/data/adonet/ef/sql-generation.md

```diff
@@ -23,7 +23,7 @@ When you write a provider for the [!INCLUDE[adonet_ef](../../../../../includes/a
 ## The Role of the SQL Generation Module  
  The SQL generation module of an [!INCLUDE[adonet_ef](../../../../../includes/adonet-ef-md.md)] provider translates a given query command tree into a single SQL SELECT statement that targets a SQL:1999-compliant database. The generated SQL should have as few nested queries as possible. The SQL generation module should not simplify the output query command tree. The [!INCLUDE[adonet_ef](../../../../../includes/adonet-ef-md.md)] will do this, for example by eliminating joins and collapsing consecutive filter nodes.  
   
- The <!--zz<xref:System.Data.Common.DBProviderServices> --> `System.Data.Common.DBProviderServices`  class is the starting point for accessing the SQL generation layer to convert command trees into <!--zz<xref:System.Data.Common.DbCommands>--> `System.Data.Common.DbCommands`.  
+ The <xref:System.Data.Common.DBProviderServices> class is the starting point for accessing the SQL generation layer to convert command trees into <xref:System.Data.Common.DbCommands>.  
```

Comment:
The reason why they were commented out was because there was something wrong with the links before. So in this case for example, DBProviderServices should be DbProviderServices.
There is no DbCommands class, so probably it should be DbCommand. (You could add a noun after to create the plural like DbCommand objects)
I'll go through them all soon to help you with this.

Reply:
Ah, I see! After we've fixed these,  I'll make sure to test if the links actually reference to existing pages in the next batches.